src/units/attack_type.hpp

```diff
@@ -87,6 +86,10 @@ class attack_type : public std::enable_shared_from_this<attack_type>
 	                      unsigned & max_attacks) const;
 	/// Returns the damage per attack of this weapon, considering specials.
 	int modified_damage(bool is_backstab) const;
+	/// Returns list for weapon like abilities
```

Comment:
still more documentation, what is the parameter? is the the tagname or the id? which filters are condifered at this point?